tf_fastmri_data/h5.py

```diff
@@ -4,15 +4,15 @@
 import tensorflow_io as tfio
 
 
-def load_data_from_file(fpath, slice_random=False, no_kspace=False, multicoil=False, mode='train'):
+def load_data_from_file(fpath, slice_num=-1, slice_random=False, no_kspace=False, multicoil=False, mode='train'):
```

Comment:
let's not call it `slice_num` because you in `load_metadata_from_file` you use the same var name for a different context.
maybe `slice_index` instead.

Reply:
Ahh yes

Reply:
Also, this would mean we would have to encode select random slice under `slice_index=-2` say. We ok with that i hope?